### PR TITLE
Gemini Exchange Order Functionalities

### PR DESCRIPTION
**Pull request by Rajat-S-Patel**

### CODE REVIEW

File: src/main/java/com/pirimid/cryptotrade/helper/exchange/gemini/ExcGemini.java [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    private long getNonce(){
+        return new Date().getTime();
+    }
+    private byte[] getB64(String payload){
+        return Base64.getEncoder().encode(payload.getBytes(StandardCharsets.UTF_8));
+    }
+    private ResponseEntity<String> apiCaller(String uri,String reqType) throws IOException, InterruptedException {
+        HttpRequest request = HttpRequest.newBuilder()
+                .uri(URI.create(uri))
+                .setHeader("X-GEMINI-APIKEY","master-sIFFhvWpYBof8QH2XECh")
```
Comment (Mahendrasinh2401) [inline src/main/java/com/pirimid/cryptotrade/helper/exchange/gemini/ExcGemini.java:46]: why fixed api key?

Comment (Rajat-S-Patel) [reply src/main/java/com/pirimid/cryptotrade/helper/exchange/gemini/ExcGemini.java:46]: removed, was not necessary for open endpoints

File: src/main/java/com/pirimid/cryptotrade/helper/exchange/gemini/ExcGemini.java [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+            String signature = GeminiUtil.getSignature(b64,secretKey);
+            ResponseEntity<String> res = apiCaller(baseUrl+"/v1/order/new","POST",b64,signature,apiKey);
+
+            CreateOrderResponse orderResponse = new ObjectMapper().readValue(res.getBody(), CreateOrderResponse.class);
+            return GeminiUtil.getPlaceOrderResDTO(orderResponse);
+        }
+        catch (NoSuchAlgorithmException | InvalidKeyException | IOException | InterruptedException e) {
+            e.printStackTrace();
+        }
+        return null;
```
Comment (HM1868) [inline src/main/java/com/pirimid/cryptotrade/helper/exchange/gemini/ExcGemini.java:120]: Can we have some proper return value instead of null?

Comment (Rajat-S-Patel) [reply src/main/java/com/pirimid/cryptotrade/helper/exchange/gemini/ExcGemini.java:120]: If null then we can send appropriate response from controller

**Pull request merged**